app/config/editor-profiles.js

```diff
@@ -77,5 +77,15 @@ export default {
     "rdfa-editor-roadsign-hint-plugin"
 
   ],
+  newFlowProfile: [
```

Comment:
please provide a decent name for the profile, also consider whether it can be merged with the drafts profie